BoardingPass/BoardingNavigationController.swift

```diff
@@ -83,44 +83,43 @@ public class BoardingNavigationController: UINavigationController {
     public var animatedTransitioningProvider: ((UINavigationControllerOperation) -> UIViewControllerAnimatedTransitioning)? = HorizontalSlideAnimatedTransiton.init
 
     public override func viewDidLoad() {
-        super.viewDidLoad()
-        delegate = self
-        configure(gestureRecognizer: panGestureRecognizer, action: #selector(handlePan))
+    super.viewDidLoad()
+    delegate = self
+    configure(gestureRecognizer: panGestureRecognizer, action: #selector(handlePan))
+    }
     }
-}
 
 #endif
 
 // MARK: - Navigation Delegate
 
 #if swift(>=3.0)
 
-extension BoardingNavigationController: UINavigationControllerDelegate {
+    extension BoardingNavigationController: UINavigationControllerDelegate {
 
-    public func navigationController(_ navigationController: UINavigationController,
-                                     animationControllerFor operation: UINavigationControllerOperation,
-                                     from fromVC: UIViewController,
-                                     to toVC: UIViewController) -> UIViewControllerAnimatedTransitioning? {
-        return animatedTransitioningProvider?(operation)
-    }
+        public func navigationController(_ navigationController: UINavigationController,
+                                         animationControllerFor operation: UINavigationControllerOperation,
+                                         from fromVC: UIViewController,
+                                         to toVC: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+            return animatedTransitioningProvider?(operation)
+        }
 
-    public func navigationController(_ navigationController: UINavigationController, interactionControllerFor animationController: UIViewControllerAnimatedTransitioning) -> UIViewControllerInteractiveTransitioning? {
-        return interactionController
+        public func navigationController(_ navigationController: UINavigationController, interactionControllerFor animationController: UIViewControllerAnimatedTransitioning) -> UIViewControllerInteractiveTransitioning? {
+            return interactionController
+        }
     }
-}
-
 
 #else
 
-extension BoardingNavigationController: UINavigationControllerDelegate {
+    extension BoardingNavigationController: UINavigationControllerDelegate {
```

Comment:
formatting is what the xCode autoindenter thinks is is correct, due to the compiler conditional (it's ugly, but I tend to not try to fight with the autoindenter)